python/Pipfile

```diff
@@ -3,10 +3,20 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 name = "pypi"
 
+[[source]]
+url = "https://rospypi.github.io/simple"
```

Comment:
There's a big warning on this repo that says "THIS IS NOT THE OFFICIAL ROSPY INDEX SERVER", what's the deal with that?

Reply:
Yes - ROS does not officially support any pip package, their installation story for python code is either through `.deb` packages or source installs via catkin or ament.